cumulusci/tasks/release_notes/parser.py

```diff
@@ -307,4 +310,24 @@ def render(self, existing_content=""):
                     f"{SANDBOX_LOGIN_URL}/packaging/installPackage.apexp?p0={version_id}",
                 ]
             )
+        if release_info is True:
+            existing_content += "\r\n".join(
+                [
+                    self._render_header(),
+                    "\r\n# Installation Info",
```

Comment:
This parser already has the title "Installation Links" (look in cumulusci.yml where the parser is configured) which is rendered by the call to _render_header(). That title should be changed to Installation Info and this line should be left out.

Reply:
So I changed https://github.com/SFDO-Tooling/CumulusCI/blob/master/cumulusci/cumulusci.yml#L1121 to Install Info, but not seeing this show up. Slightly unsure about this, will push a proposed way to handle see what you think.

Reply:
This part has me quite confused, I've made the change locally but my tests are not giving the desired result.